governance/roles.md

```diff
@@ -202,41 +209,104 @@ that role for some time informally, then proposed as an on-going role by Pull
-
-* coordinate security review leads for upcoming security assessments.
-* identify and recommend security reviewers.
-* contribute to process improvements.
-* review and merge PRs in the /assessments directory (ensuring co-chair review
-  of significant process changes).
-* triage issues related to security assessments.
-
-### Meeting facilitator
+and led by a security assessment facilitator (referred to in the rest of this
+sub-section as a facilitator). The facilitator is effectively responsible for
+getting new security assessments underway, and providing support on an as-needed
+basis to the security assessment team if anything is impeding the overall review
+process.
+
+The facilitator is responsible for coordinating with the TAG-Security co-chairs
+and deciding on the order in which security assessments (as submitted in the
+[GitHub issues tracker for
+TAG-Security](https://github.com/cncf/tag-security/issues?q=is%3Aopen+is%3Aissue+label%3Aassessment)
+will be addressed.  The facilitator is then responsible for reaching out to the
+project owners (via the GitHub issue said owners previously created in order to
+request an assessment), and coordinating the various requirements as outlined
+in the ["Joint Security Review" ticket
+template](https://github.com/cncf/tag-security/issues/new/choose). Once the
+maintainers/owners of the project have been identified, the facilitator is
+responsible for reaching out to the TAG-Security community at large (i.e.
+typically via attending the weekly general meetings or announcing to the
+community [via the `#tag-security`
+channel](https://cloud-native.slack.com/archives/CDJ7MLT8S)), and issue a
+call/request for reviewers (i.e. a single security assessment lead, and at
+least two additional security reviewers), and ensuring all reviewers [read the
+conflict of interest disclosure](../assessments/guide/security-reviewer.md) and
+sign-off on it in the GitHub ticket itself. The sign-off is a simple form that
+can be easily copy/pasted into a new reply in the GitHub issue/ticket
+associated with the assessment (and the individual replaces the `???` literal
+text with their response), e.g.:
+
+```txt
+Hard conflicts:
+Reviewer is a maintainer of the project - ???
+Reviewer is a direct report of/to a maintainer of the project - ???
+Reviewer is paid to work on the project - ???
+Reviewer has significant financial interest directly tied to success of the project - ???
+
+Soft conflicts:
+Reviewer belongs to the same company/organization of the project, but does not work on the project - ???
+Reviewer uses the project in his/her work - ???
+Reviewer has contributed to the project. - ???
+Reviewer has a personal stake in the project (personal relationships, etc.) - ???
+```
+
+From this point forward, the security assessment lead will be the primary
+individual most responsible for driving the security assessment alongside the
+security reviewers, as well as the project owners.
```

Comment:
Mention here that facilitator will act as a contact point for escalation as well as to do regular check-ins to ensure project is on track.